src/llmq/quorums.cpp

```diff
@@ -1003,18 +1003,24 @@ void CQuorumManager::CleanupOldQuorumData(const CBlockIndex* pIndex) const
         return;
     }
 
-    std::set<uint256> dbKeys;
+    std::set<uint256> dbKeysToSkip;
 
     LogPrint(BCLog::LLMQ, "CQuorumManager::%d -- start\n", __func__);
+    // Platform quorums in all networks are newed every 24h (LLMQ_100_67, LLMQ_25_67, LLMQ_DEVNET_PLATFORM and LLMQ_REGTEST_PLATFORM)
+    // During a month, 24 x 30 quorums are created
+    // We want to keep data for Platform quorums no older than 2 months since Platform can be restarted and needs to re-sign stuff
+    // Hence, we should not clean secret key shares and vvec for those quorums.
+    constexpr auto numPlatformQuorumsDataToKeep = 24 * 30 * 2;
```

Comment:
>Platform quorums in all networks are newed every 24h (LLMQ_100_67, LLMQ_25_67, LLMQ_DEVNET_PLATFORM and LLMQ_REGTEST_PLATFORM)

that's not true I think

```suggestion
    // Platform quorums in all networks are created every 24 blocks (~1h).
    // Unlike for other quorum types we want to keep data (secret key shares and vvec)
    // for Platform quorums for at least 2 months because Platform can be restarted and
    // it must be able to re-sign stuff. During a month, 24 * 30 quorums are created.
    constexpr auto numPlatformQuorumsDataToKeep = 24 * 30 * 2;
```

Reply:
@UdjinM6 `LLMQ_50_60`, `LLMQ_100_67`, `LLMQ_25_67`, `LLMQ_DEVNET_PLATFORM`, `LLMQ_TEST_PLATFORM` have 24 `dkgInterval`.
Unless if you are referring to the cases where we overwrite `llmqTypePlatform` with another llmq.

Reply:
>    // The interval in number blocks for DKGs and the creation of LLMQs. If set to 24 for example, a DKG will start
>    // every 24 blocks, which is approximately once every hour.

`dkgInterval` has nothing to do with 24h

Reply:
Sorry, I meant 24 blocks. But yeah, we are aligned :)